lib/recommended-rules.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   "ember/alias-model-in-controller": "off",
   "ember/avoid-leaking-state-in-components": "off",
   "ember/avoid-leaking-state-in-ember-objects": "error",
+  "ember/avoid-using-needs-in-controllers": "error",
```

Comment:
We cannot update the recommeded rule set without a major version bump. Please remove this change for now...